### PR TITLE
Add base functionality

### PR DESCRIPTION
**Pull request by thc202**
Run the provided Automation Framework plan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+
+## Inputs
+
+### `plan`
+
+**Required** The file system path to the Automation Framework plan to run.
+
+### `docker_name`
+
+**Required** The name of the [ZAP Docker image](https://www.zaproxy.org/docs/docker/about/#install-instructions) to be used. By default the action runs the stable image.
```
Comment (kingthorin) [inline README.md]: This is confusing, it's required but there's a default?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+
+### `cmd_options`
+
+**Optional** Additional [command lines options](https://www.zaproxy.org/docs/desktop/cmdline/) for ZAP.
```
Comment (kingthorin) [inline README.md]: Singular "line"

File: README.md (hunk 1 above)
Comment (kingthorin) [reply README.md]: If there's a default why require setting it... (make it optional if they want something other than default)

Comment (thc202) [reply README.md]: You don't need to specify it but if you do it must not be empty, that's why the required. Would "mandatory" be better? Also happy to say "optional".

Comment (kingthorin) [reply README.md]: Optional, if specified must not be empty? Or if specified but empty the default (stable) is used...

Something like that. Maximize user friendliness 👍

Comment (thc202) [reply README.md]: Changes pushed.

**Review by kingthorin — APPROVED** [review]
LGTM

**Review by psiinon — APPROVED** [review]

**Pull request merged**